PROTO_tests/tests/numeric_test.py

```diff
@@ -0,0 +1,362 @@
+import arkouda as ak
+import pytest
+import numpy as np
+
+from arkouda.dtypes import npstr
+
+NUMERIC_TYPES = [ak.int64, ak.float64, ak.bool, ak.uint64]
+CAST_TYPES = [ak.int64, ak.float64, ak.bool, ak.uint64, npstr]
```

Comment:
`CAST_TYPES` seems to be parametrization of this inner loop in `testCast`
```python
for t2 in ak.DTypes:
    t2 = ak.dtype(t2)
```

But the inner loop seems to cover more types than those listed, most notably `bigint` is missing
```python
In [4]: for t2 in ak.DTypes:
   ...:     t2 = ak.dtype(t2)
   ...:     print(t2)
   ...:
bigint
int64
float64
uint64
<U0
bool
float64
uint8
int64

In [5]: ak.DTypes
Out[5]:
frozenset({'bigint',
           'bool',
           'float',
           'float64',
           'int',
           'int64',
           'str',
           'uint64',
           'uint8'})
```

Reply:
Updated to use the list comprehension @pierce314159 noted above.